hcn/hcnglobals.go

```diff
@@ -25,6 +25,8 @@ var (
 	HNSVersion1803 = Version{Major: 7, Minor: 2}
 	// V2ApiSupport allows the use of V2 Api calls and V2 Schema.
 	V2ApiSupport = Version{Major: 9, Minor: 1}
+	// Remote Subnet allows for Remote Subnet policies on Overlay networks
+	RemoteSubnetVersion = Version{Major: 9, Minor: 2}
```

Comment:
Why do we need this? Seems like the `GetSupportedFeatures().RemoteSubnet == true` tells us

Reply:
Oh I see your not asking the product. Can we query the hns for this?

Reply:
HNS is queried in GetSupportedFeatures() and then  the version is compared to this

Reply:
Ok no prob.